assets/css/styleguide.scss

```diff
@@ -831,12 +734,20 @@ h3 + .button_wrapper {
   padding-bottom: .8rem;
 }
 
+// scss-lint:disable QualifyingElement
 h6.usa-heading-alt {
   margin-top: 4rem;
 }
+// scss-lint:enable QualifyingElement
 
 .usa-typography-example-font {
-  h1, h2, h3, h4, h5, h6, .usa-font-example p {
+  h1,
```

Comment:
@msecret sweet, should I revert this change and disable the `SingleLinePerSelector` linter, or...?